qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -121,21 +139,98 @@ def excitations(
+        return self._gsc.qubit_converter
+
     @property
     def solver(self):
+        """Returns the solver object defined in the ground state solver."""
         return self._gsc.solver
 
     def get_qubit_operators(
         self,
         problem: BaseProblem,
-        aux_operators: Optional[dict[str, Union[SparseLabelOp, PauliSumOp]]] = None,
-    ) -> Tuple[PauliSumOp, Optional[dict[str, PauliSumOp]]]:
+        aux_operators: Optional[Dict[str, Union[SparseLabelOp, PauliSumOp]]] = None,
+    ) -> Tuple[PauliSumOp, Optional[Dict[str, PauliSumOp]]]:
+        """Construct qubit operators by getting the second quantized operators from the problem
+        (potentially running a driver in doing so [can be computationally expensive])
+        and using a QubitConverter to map and reduce the operators to qubit operators.
+
+        Args:
+            problem: A class encoding a problem to be solved.
+            aux_operators: Additional auxiliary operators to evaluate.
+
+        Returns:
+            Qubit operator.
+            Additional auxiliary operators.
+        """
         return self._gsc.get_qubit_operators(problem, aux_operators)
 
+    def get_qeom_qubit_operators(
+        self,
+        problem: BaseProblem,
+        aux_operators: Optional[Dict[str, Union[SparseLabelOp, PauliSumOp]]] = None,
+    ) -> Tuple[
+        QubitOperator, Optional[Dict[str, QubitOperator]], Optional[Dict[str, QubitOperator]]
+    ]:
+        """Gets the operator and auxiliary operators, and transforms the provided auxiliary operators.
+        Note that contrary to the method :meth:`get_qubit_oprators` from the
+        :class:`GroundStateEigensolver`, this returns three outputs: the hamiltonian, second quantization
+        default auxiliaries, and user defined auxiliaries.
+        Also note that this methods performs a specific treatment of the symmetries required by the qEOM
+        calculation."""
+
+        main_second_q_op, aux_second_q_ops = problem.second_q_ops()
+
+        # 1. Convert to PauliSumOp and apply two qubit reduction
+        # We apply the meth:`convert()` with the symmetries deliberately set to None
+        num_particles = getattr(problem, "num_particles", None)
+        main_operator = self.qubit_converter.convert_only(
+            main_second_q_op,
+            num_particles=num_particles,
+        )
+        self.qubit_converter.force_match(num_particles=num_particles)
+
+        aux_default_ops = self.qubit_converter.convert_match(aux_second_q_ops)
+        aux_custom_ops = {}
+        if aux_operators is not None:
+            for name_aux, aux_op in aux_operators.items():
+                if isinstance(aux_op, (SparseLabelOp)):
+                    converted_aux_op = self.qubit_converter.convert_match(aux_op)
+                else:
+                    converted_aux_op = aux_op
+
+                aux_custom_ops[name_aux] = converted_aux_op
```

Comment:
One issue is that the GroundstateEigensolver returns an error when it is passed a dictionary of aux operators containing the default second_q operators. Else I would also prefer treating them simultaneously

Reply:
Oh because of clashing names? Where exactly do you pass these on?

I'm assuming that you provide these via the `aux_operators` to the ground-state solver and then it complains because it also gets the default ones out of the problem. If so I would rather add a precedence logic which means that custom aux operators will overwrite the built-in ones if the names collide.